libs/sync/src/assets/i18n/en.json

```diff
@@ -82,6 +82,10 @@
 	"sync.details.buttons.edit-settings": "Edit settings",
 	"sync.details.buttons.save-changes": "Save changes",
 	"sync.details.buttons.sync-now": "Sync now",
+	"sync.details.modal.description": "You can either synchronize the new data created or modified since the last synchronization, or resynchronize all the data from the beginning.",
```

Comment:
I feel like `and` would be better than `or` in `the new data created or modified since the last synchronization`